app/models/sushi.rb

```diff
@@ -151,6 +133,10 @@ def coerce_dates(params = {})
       # Because we're receiving user input that is likely strings, we need to do some coercion.
       @begin_date = Sushi.coerce_to_date(params.fetch(:begin_date)).beginning_of_month
       @end_date = Sushi.coerce_to_date(params.fetch(:end_date)).end_of_month
+
```

Comment:
since we're here...can we also account for when the begin or end date fall outside of the dates we have data for? (ref: #764)

Reply:
This is a significant and chunky change to the specs; I have the written production logic stenciled in but it seriously breaks a lot of specs.

My strong preference would be to merge this and then build on that requirement.